### ISSUE TITLE
Is VTODO items are / will be supported ?

### ISSUE THREAD

**Issue by CharlesHD** [dengste/org-caldav#71]
my org-icalendar exports TODO entries into VTODO icalendar items wich are not synced (but the sync doesn't fail).
In the other hand if my ownCloud calendar have tasks, the sync fail here's the backtrace : 

``` elisp
  signal(error ("Error while getting eventlist from https://owncloud.serv/remote.php/caldav/calendars/charleshd/tasks/."))
  error("Error while getting eventlist from %s." "http://owncloud.serv/remote.php/caldav/calendars/charleshd/tasks/")
  org-caldav-get-event-etag-list()
  org-caldav-update-events-in-cal(#<buffer org-caldav-109500Bs>)
  org-caldav-sync-calendar((:calendar-id "tasks" :files ("~/org/taskdiary.org") :inbox "~/org/fromtask.org"))
  org-caldav-sync()
  call-interactively(org-caldav-sync record nil)
```

Well i suspect VTODO items aren't supported but it would be a cool feature.

**Comment by hugoroy** [comment]
Yes, support for VTODO would be awesome :-)

**Comment by grauschnabel** [comment]
see #118

**Comment by grauschnabel** [comment]
this is now supported but not yet merged here.  you can use my org-caldav repository if you need that. :)

**Comment by CharlesHD** [comment]
Nice thanks you.
I'll close the issue once @dengste will pronounce on merging your PR or not.

**Comment by pierrebonbon** [comment]
Any news on the merge? This looks like an awesome feature.

**Comment by grauschnabel** [comment]
Yes, but I hade an issue with formating.  So I cannot really go back. Any linting form @dengste would be fine to autoformat the code in the lisp files.

**Comment by jackkamm** [comment]
Resolved by #218
